MachObfuscator/DependencyAnalysis/ObfuscableFilesFilter.swift

```diff
@@ -11,6 +11,21 @@ extension ObfuscableFilesFilter {
         }
     }
 
+    func negate() -> ObfuscableFilesFilter {
+        return ObfuscableFilesFilter { !self.isObfuscable($0) }
+    }
+
+    func whitelist(_ other: ObfuscableFilesFilter) -> ObfuscableFilesFilter {
```

Comment:
Do we want this to be `whitelist`, or could we just change it to be `or`?

Reply:
this is "special" `or`: `other` is evaluated before `self`, not after as would be expected from ordinary `or`

Reply:
renamed